index.js

```diff
@@ -133,25 +140,10 @@ Object.assign(updater, {
   },
 
   updateTaskDefinitionImage(taskDefinition, containerNames, image) {
```

Comment:
now that this is not mutating, maybe a good idea to rename to something that sounds non mutatey, or just move it up into the function up above